ui/app/components/tools/wrap.js

```diff
@@ -24,15 +25,31 @@ export default class ToolsWrap extends Component {
   @tracked buttonDisabled = false;
   @tracked token = '';
   @tracked wrapTTL = null;
-  @tracked wrapData = '{\n}';
+  @tracked wrapData;
   @tracked errorMessage = '';
+  @tracked showJson = true;
+  get jsonWrapStartValue() {
```

Comment:
Getters are useful for computing return values. we're just returning a static string here so this can just be a variable.